app/models/user.rb

```diff
@@ -1,4 +1,14 @@
 class User < ActiveRecord::Base
+  belongs_to :neighborhood
+  belongs_to :building
+
+  validates_presence_of :first_name
+  validates_presence_of :last_name
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.